lib/timers.js

```diff
@@ -377,13 +377,23 @@ const unenroll = exports.unenroll = function(item) {
   }
   // if active is called later, then we want to make sure not to insert again
   item._idleTimeout = -1;
+}
+
+exports.unenroll = function(item) {
+  process.emitWarning('timers.unenroll is deprecated. ' +
```

Comment:
Will this not emit the warning every time it is called vs just the first time? Any reason to not use util.deprecate() for this?

Reply:
good catch.

Reply:
Also, message should probably include `()` after the function name: `timers.unenroll() is deprecated.`